packages/components/src/font-size-picker/index.js

```diff
@@ -118,6 +148,16 @@ export default function FontSizePicker( {
 					{ __( 'Reset' ) }
 				</Button>
 			</div>
+			<div>
+				<ToggleControl
+					label="Advanced font sizing"
```

Comment:
Text should be translatable

```suggestion
					label={ __( 'Advanced font sizing' ) }
```